wordpress-mobile/WordPress-Android
# Track analytic event when editor help is shown

## Related PRs

* https://github.com/wordpress-mobile/WordPress-iOS/pull/17045

## Description

Improve our ability to understand how users interact with the editor help content.

To test:

1. Launch the editor.
1. Tap the three-dot button in the top right corner of the editor.
1. Tap Help.
1. ℹ️ **Expected:** The `wpandroid_editor_help_shown` event arrives in Tracks.

## Regression Notes
1. Potential unintended areas of impact
  The new analytic event could disrupt opening the help content.
2. What I did to test those areas of impact (or what existing automated tests I relied on)
  Verified the help content continues to display.
3. What automated tests I added (or what prevented me from doing so)
  The majority of the help content logic resides in the editor, so placing tests
within this repository makes less sense.

PR submission checklist:
